cmd/terramate/e2etests/exp_vendor_test.go

```diff
@@ -132,6 +135,104 @@ func TestVendorModuleRecursive1DependencyIsPatched(t *testing.T) {
 	assert.EqualStrings(t, "", string(got))
 }
 
+func TestModVendorRecursiveMustPatchAlreadyVendoredModules(t *testing.T) {
```

Comment:
this seems unrelated to the PR no ? :thinking:

Reply:
this PR has nothing to do with vendoring, this was showing here probably because the branch wasn't updated with the main branch...

Reply:
ahhh OK :laughing: